src/docs/getting-started/usage.mdx

```diff
@@ -0,0 +1,136 @@
+performance).
+
+## Foundation CSS
+
+React UI honors ITCSS principles to make sure that its CSS code base will work
+and perform well even in large scale. There are two global CSS layers:
+
+- **Theme:** a collection of hundreds of CSS custom properties that define the
+  look of your app. See [Theming](/getting-started/theming) for more.
+- **Foundation:** the ground-zero CSS for React UI components. Includes global
+  resets and fixes rendering inconsistencies across browsers with
+  `normalize.css`. Not to be confused with the Foundation CSS framework.
+
+Import the two global CSS layers as early as possible in your document:
+
+```js
+import '@react-ui-org/react-ui/src/lib/theme.scss';
+import '@react-ui-org/react-ui/src/lib/foundation.scss';
+```
+
+⚠️ As of now, you will need **working SASS pipeline** for this to work. This is
+to be changed to a ready-to-use CSS in the future.
+
+️👉️ Because of using CSS custom properties, it is technically possible to import
+the theme layer anywhere later in the app. But it's a nice habit to declare
+variables before using them.
+
+## Components
+
+Import and use any of React UI components in your app:
+
+```jsx
+import { Button } from '@react-ui-org/react-ui';
+
+<Button label="My Button" />
+```
+
+## Full Example
+
+Example HTML:
+
+```html
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>React UI Example</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300;400;600;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="app.js"></script>
+  </body>
+</html>
+```
+
+Example JSX:
+
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Button } from '@react-ui-org/react-ui';
+import '@react-ui-org/react-ui/src/lib/theme.scss';
+import '@react-ui-org/react-ui/src/lib/foundation.scss';
+
+const App = () => (
+  <Button label="My Button" />
+);
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+## Helpers and Utilities
+
+Helpers and utilities are tiny CSS classes that can handle details like
+[typography](/utilities/typography), [spacing](/utilities/spacing),
+[colors](/utilities/colors), etc.
+
+While the **utility classes do just one thing** (they set a single CSS rule and
+use the otherwise disallowed `!important` to enforce it) and are often
+responsive (can be adjusted for individual breakpoints), the **helper classes
+can compose multiple CSS rules** to do a bit more complicated tasks.
+
+To use helpers and utilities, just import them along the base CSS layers of
+React UI:
+
+```js
+import '@react-ui-org/react-ui/src/lib/theme.scss';
+import '@react-ui-org/react-ui/src/lib/foundation.scss';
+import '@react-ui-org/react-ui/src/lib/helpers.scss';
+import '@react-ui-org/react-ui/src/lib/utilities.scss';
+```
+
+The class name notation is
```

Comment:
I don't think so. Firstly it's a "selling point" (we use something what somebody else uses too), secondly it's better to admit the source of inspiration rather than just copying it.